### DISCUSSION TITLE
Tab Delimited File : Stop Quoting Fields with Quote in the Data (and stop Escaping data's Quote)

### DISCUSSION THREAD

**Discussion by chadbruels** [JoshClose/CsvHelper#2224] [Q&A] [answered]
```
csv.Configuration.Delimited = "\t";
csv.Configuration.HasHeaderRecord = false;
csv.WriteRecords(poLines);
```

I'm creating a tab delimited file with the code above and the consuming party does not want data quoted (and as a by-product not escaped either). I have description fields which have double quotes in the data. How do I stop CsvHelper from automatically wrapping the field in quote and escaping the double quote in the data? Is this done at a field-by-field basis or can I specify one config value to apply to all fields in the file?

**Comment by JoshClose** [comment]
You can either set `ShouldQuote = args => false` or `Mode = CsvMode.NoEscape`

**Reply by chadbruels** [reply]
Thanks, Josh.
What if the project is using v16.0 of CsvHelper and I was instructed to continue using that version?

**Reply by JoshClose** [reply, marked as answer]
`ShouldQuote = (_, _) => false`

or if old C# lang too

`ShouldQuote = (field, context) => false`
